sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -73,6 +73,8 @@ function createAssetClaimInfoSerialized(hash)
  * First/second step of a Hashed Time Lock Contract
  * - Lock a unique asset instance using a hash
  **/
+//Byzantine Swaps: Call StartHTLCAssetLockListener within here while passing new callback 
+//parameter as newly defined function: assetLockCallbackTime is passed as localCallback
```

Comment:
Add these comments to the ones in 73-75 instead of placing them separately.

Reply:
Also name the callback function accurately. I'd prefer to call it `assetLockExpirationCallback`. (See further below.)